CHANGES/1095.contrib.rst

```diff
@@ -0,0 +1 @@
+Replace all :type:`Optional` with :type:`Union` -- by :user:`bdraco`.
```

Comment:
```suggestion
Replaced all :type:`Optional` with :type:`Union` -- by :user:`bdraco`.
```

Oh, I didn't realize you can link types like that..

Reply:
It was an attempt but actually I don't think you can.

```
/home/docs/checkouts/readthedocs.org/user_builds/yarl/checkouts/1095/docs/changes.rst:10: WARNING: py:type reference target not found: Optional [ref.type]
/home/docs/checkouts/readthedocs.org/user_builds/yarl/checkouts/1095/docs/changes.rst:10: WARNING: py:type reference target not found: Union [ref.type]
```

Reply:
Oh, okay. That's what I assumed :)

You'd need to declare the corresponding role. You need to go for 
```rst
:class:`~typing.Optional`
```

In general, the best way to know if an object exists is to call intersphinx from command line or view as html on my helper static site..

Reply:
@bdraco missed this comment with the working syntax?

Reply:
Yup. Just saw it now

Reply:
https://github.com/aio-libs/yarl/actions/runs/10746477913 is still going.  If it gets past the wheel combine this time, I'll cancel it and rework the changelog message and start the release process over

Reply:
https://github.com/aio-libs/yarl/pull/1121